src/shared/components/app-icon/GenomeBrowserIcon.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+
+import { ReactComponent as SVGIcon } from 'static/img/launchbar/browser.svg';
```

Comment:
for consistency purpose, should the svg not be called genome-browser.svg

Reply:
🤷‍♂️  
We'll figure it out in [this ticket](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1456), when we organise our svg icons. Andrea's name of the file was different still (`icon_launchbar_browser.svg`). We'll worry about that then.